setup.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
 from setuptools import find_packages, setup
+# from setuptest import test
```

Comment:
no commented lines like this please

Reply:
ah sorry, was meant to remove that, thanks!

Reply:
👍